front/app/index.html

```diff
@@ -18,11 +18,17 @@
     <link rel="stylesheet" href="assets/vendor.css">
     <link rel="stylesheet" href="assets/rust-dashboard.css">
 
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/github-fork-ribbon-css/0.2.0/gh-fork-ribbon.min.css" />
+    <!--[if lt IE 9]>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/github-fork-ribbon-css/0.2.0/gh-fork-ribbon.ie.min.css" />
+    <![endif]--> 
+
     {{content-for "head-footer"}}
   </head>
   <body>
     {{content-for "body"}}
-
+    
+    <noscript>Please enable javascript to view this website.</noscript>
```

Comment:
This is a good addition to ensure those users aren't just presented with a blank page!